server/src/unifyfs_group_rpc.c

```diff
@@ -479,9 +462,10 @@ int invoke_bcast_progress_rpc(coll_request* coll_req)
     }
 
     /* get handle to local rpc function */
+    hg_handle_t handle;
```

Comment:
Why did this change to a local variable? Revert to using the handle saved in the the coll_request

Reply:
fixed